server.js

```diff
@@ -21,7 +20,37 @@ app.use(express.json());
 
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  const endpoints = expressListEndpoints(app);
+  res.send(endpoints);
+});
```

Comment:
if your endpoint accepts some queries to filter the data, it's good to inform the user and add it to the documentation. You could do something like this:

```javascript
app.get("/", (req, res) => {
  const endpoints = expressListEndpoints(app);
  const updatedEndpoints = endpoints.map((endpoint) => {
    if (endpoint.path === '/netflix-titles') {
      endpoint.query = ['release_year']; // Add the release_year query parameter
    }
    return endpoint;
  });
  res.json(updatedEndpoints);
});
```